src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+class ColorSupplier {
+    public static String getRandomColor() {
```

Comment:
The method `getRandomColor` should not be static according to the checklist. It's better to create an instance of the `ColorSupplier` class and call the method on it.